users/storage.go

```diff
@@ -17,6 +17,15 @@ type StorageBackend interface {
 	DeleteByUsername(string) error
 }
 
+type Interface interface {
```

Comment:
Please give a proper name to this interface. `UserStore`?

Reply:
The linter will refuse that due to stuttering, package symbols should not start with the package name. This ends up being used as `users.Interface` which IMHO is totally reasonable

Reply:
I just gave an example. It can be named Store. An interface name should be meaningful and self explainable. `Interface` name is too generic.

Reply:
Renamed it to `Store`, please take another look